packages/modal/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "main": "lib/index.js",
```

Comment:
This might be an outdated package.json generation. Can you please take a look at another hook in the `packages` and look at the package.json fields. It was changed fairly recently.

Reply:
Rebased to your `dev` branch. Should be ok now.